tests/10-issue.js

```diff
@@ -64,206 +69,248 @@ describe('Issuers - BitstringStatusList',
                     'message').that.is.a('string');
-                });
               } else {
-                it.skip('If statusSize is not present as a property ' +
-                  'of the credentialStatus, then statusSize MUST ' +
-                  'be processed as 1.',
-                async function() {
-                  this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=If%20statusSize%20is%20not%20present%20as%20a%20property%20of%20the%20credentialStatus%2C%20then%20statusSize%20MUST%20be%20processed%20as%201.';
-                }
-                );
+                this.skip();
               }
-            });
-          it('An implementer MAY include the statusReference property.',
-            async function() {
-              this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=An%20implementer%20MAY%20include%20the%20statusReference%20property.';
-              if('statusReference' in issuedVc.credentialStatus) {
-                it('If present, its value MUST be a URL or an array of URLs ' +
-                    '[URL] which dereference(s) to material related to the ' +
-                    'status.',
-                async function() {
-                  this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=If%20present%2C%20its%20value%20MUST%20be%20a%20URL%20or%20an%20array%20of%20URLs%20%5BURL%5D%20which%20dereference%20to%20material%20related%20to%20the%20status';
-                  const statusReferenceType = typeof (
-                    issuedVc.credentialStatus.statusReference);
-                  statusReferenceType.should.be.oneOf(['string', 'object'],
-                    'Expected statusReference to be an string or an array.');
-                  if(statusReferenceType === 'object') {
-                    issuedVc.credentialStatus.statusReference.should.be.an(
-                      'array');
-                    issuedVc.credentialStatus.statusReference.forEach(
-                      item => item.should.be.a('string'));
-                  }
-                  // TODO test for URLS
-                });
+            }
+          });
+          it('If present, the "statusReference" value MUST be a URL or an ' +
+              'array of URLs [URL] which dereference(s) to material related ' +
+              'to the status.',
+          async function() {
+            this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=If%20present%2C%20its%20value%20MUST%20be%20a%20URL%20or%20an%20array%20of%20URLs%20%5BURL%5D%20which%20dereference%20to%20material%20related%20to%20the%20status';
+            for(statusEntry in statusEntries) {
+              if('statusReference' in statusEntries[statusEntry]) {
+                const statusReferenceType = typeof (
+                  statusEntries[statusEntry].statusReference);
+                statusReferenceType.should.be.oneOf(['string', 'object'],
```

Comment:
this is cool, but maybe should be a shared assertion like `shouldBeStringOrObject` or something like that.

Reply:
I've opted to leave it like this since this is only 1 line of code and removes the ability to write a specific error message

Reply:
i like the reasoning there so resolving this. good work on this PR.